kube/src/config/mod.rs

```diff
@@ -90,6 +92,7 @@ impl Config {
             timeout: None,
             accept_invalid_certs: false,
             identity: None,
+            identity_password: String::new(),
```

Comment:
Yeah, should this not be IDENTITY_PASSWORD?

Reply:
It doesn't really matter what it is since we're not using it. Perhaps we should store `identity` as `Option<(Vec<u8>, String)>` since we only need the password if we have the identity buffer.